### ISSUE TITLE
Touchpad improvement with new elan kext

### ISSUE THREAD

**Issue by BAndysc** [cobaohieu/ASUS-K55VD-HACKINTOSH#31]
According to my research, you have ELAN PS/2 touchpad in the Asus laptop. For much much better (native macOS gestures), check my version of kext: https://github.com/BAndysc/VoodooPS2/releases/

**Comment by cobaohieu** [comment]
Yeah, That's a great idea. However, you will get some issue when you're controlling the touchpad with only the VoodooPS2 kext.
Please check my release to make sure it's working all the time
**https://github.com/cobaohieu/ASUS-K55VD-HACKINTOSH/releases**
